ht/cogs/roll.py

```diff
@@ -1,4 +1,8 @@
+import aiohttp
+import asyncio
+import re
 import typing
```

Comment:
```suggestion
import aiohttp, asyncio, re, typing
```

This is the novov house style™